common/src/logger/loggers/mod.rs

```diff
@@ -123,7 +139,13 @@ pub fn create_root_logger() -> Logger {
         (None, Some(udp_json)) => Some(udp_json),
         (Some(_), Some(_)) => panic!("MC_LOG_GELF and MC_LOG_UDP_JSON are mutually exclusive!"),
     };
-    let stdout_logger = create_stdout_logger();
+
+    // Create stdout / stderr sink
+    let std_logger = if env::var("MC_LOG_STDERR").is_ok() {
```

Comment:
Can you instead compare it to Ok("1")?

Reply:
we could

Reply:
done

Reply:
I wonder if there's a Rust util to parse command-line string truthiness ("1", "true", etc)?